Build fails because wrensec-ui tries to contact non-existing FR repo

Hangs for some time on `Running 'npm install --color=false'` and after a while starts spewing "fetch failed" errors:

```
[INFO] --- frontend-maven-plugin:0.0.27:npm (npm-install) @ forgerock-ui-commons ---
[INFO] Found proxies: []
[INFO] Running 'npm install --color=false' in /home/siepkes/git/wrensecurity/wrensec-ui/forgerock-ui-commons
[ERROR] npm ERR! fetch failed http://maven.forgerock.org:80/repo/api/npm/npm-virtual/grunt-cli/-/grunt-cli-0.1.13.tgz
[ERROR] npm WARN retry will retry, error on last attempt: Error: connect ECONNREFUSED 127.0.0.1:80
[ERROR] npm ERR! fetch failed http://maven.forgerock.org:80/repo/api/npm/npm-virtual/eslint-formatter-warning-summary/-/eslint-formatter-warning-summary-1.0.1.tgz
[ERROR] npm WARN retry will retry, error on last attempt: Error: connect ECONNREFUSED 127.0.0.1:80
[ERROR] npm ERR! fetch failed http://maven.forgerock.org:80/repo/api/npm/npm-virtual/eslint/-/eslint-1.5.1.tgz
[ERROR] npm WARN retry will retry, error on last attempt: Error: connect ECONNREFUSED 127.0.0.1:80
[ERROR] npm ERR! fetch failed http://maven.forgerock.org:80/repo/api/npm/npm-virtual/grunt-eslint/-/grunt-eslint-17.3.1.tgz
[ERROR] npm WARN retry will retry, error on last attempt: Error: connect ECONNREFUSED 127.0.0.1:80
[ERROR] npm ERR! fetch failed http://maven.forgerock.org:80/repo/api/npm/npm-virtual/rimraf/-/rimraf-2.5.3.tgz
[ERROR] npm WARN retry will retry, error on last attempt: Error: connect ECONNREFUSED 127.0.0.1:80

```

--- a/forgerock-ui-commons/.npmrc
+++ b/forgerock-ui-commons/.npmrc
@@ -1,1 +1,0 @@
-registry = http://maven.forgerock.org/repo/api/npm/npm-virtual/

--- a/forgerock-ui-mock/.npmrc
+++ b/forgerock-ui-mock/.npmrc
@@ -1,1 +1,0 @@
-registry = http://maven.forgerock.org/repo/api/npm/npm-virtual/

--- a/forgerock-ui-user/.npmrc
+++ b/forgerock-ui-user/.npmrc
@@ -1,1 +1,0 @@
-registry = http://maven.forgerock.org/repo/api/npm/npm-virtual/

--- a/pom.xml
+++ b/pom.xml
@@ -107,7 +107,7 @@
                 <plugin>
                     <groupId>com.github.eirslett</groupId>
                     <artifactId>frontend-maven-plugin</artifactId>
-                    <version>0.0.27</version>
+                    <version>0.0.28</version>
                     <configuration>
                         <installDirectory>${node.install.directory}</installDirectory>
                     </configuration>
@@ -119,9 +119,10 @@
                             </goals>
                             <phase>initialize</phase>
                             <configuration>
-                                <nodeVersion>v4.2.6</nodeVersion>
-                                <npmVersion>3.5.3</npmVersion>
-                                <npmDownloadRoot>http://maven.forgerock.org/repo/api/npm/npm-virtual/npm/-/</npmDownloadRoot>
+                                <nodeVersion>v4.4.5</nodeVersion>
+                                <npmVersion>3.9.3</npmVersion>
+                                <downloadRoot>https://nodejs.org/dist/</downloadRoot>
+                                <npmDownloadRoot>https://wrensecurity.jfrog.io/wrensecurity/api/npm/npm-virtual/npm/-/</npmDownloadRoot>
                             </configuration>
                         </execution>
 
@@ -131,9 +132,6 @@
                                 <goal>npm</goal>
                             </goals>
                             <phase>initialize</phase>
-                            <configuration>
-                                <arguments>install</arguments>
-                            </configuration>
                         </execution>
 
                         <execution>
@@ -143,7 +141,7 @@
                             </goals>
                             <phase>compile</phase>
                             <configuration>
-                                <arguments>build</arguments>
+                                <arguments>build --debug --verbose</arguments>
                             </configuration>
                         </execution>
                     </executions>

